From a323b51eb936e92acd57167027b50491afbee97b Mon Sep 17 00:00:00 2001
From: Donald McIntyre <donald@etherplan.com>
Date: Tue, 5 Nov 2019 19:35:35 -0700
Subject: [PATCH] Adding author and license to ECIP-1015

On behalf of the author, adding his name as author and adding a license to the ECIP.

The authorship may be confirmed here from original document: https://github.com/ethereumproject/ECIPs/blob/master/ECIPs/ECIP-1015.md

The license added is Apache 2.
---
 _specs/ecip-1015.md | 14 ++++++++++----
 1 file changed, 10 insertions(+), 4 deletions(-)

diff --git a/_specs/ecip-1015.md b/_specs/ecip-1015.md
index 5d5be52..0856aef 100644
--- a/_specs/ecip-1015.md
+++ b/_specs/ecip-1015.md
@@ -2,7 +2,7 @@
 lang: en
 ecip: 1015
 title: Long-term gas cost changes for IO-heavy operations to mitigate transaction spam attacks
-author: Unknown
+author: Cody W Burns <cody.w.burns@ethereumclassic.org>
 status: Final
 type: Standards Track
 category: Core
@@ -12,7 +12,7 @@ category: Core
 * Concerning: Long-term gas cost changes for IO-heavy operations to
   mitigate transaction spam attacks
 
-With the understanding that:
+## With the understanding that:
 
 1. This is a fork intended for protocol improvement.
 2. This is mutually beneficial to all etherem clients in order to mitigate current attacks.
@@ -21,7 +21,7 @@ With the understanding that:
 4. An unknown number of ETC nodes are using --oppose-dao clients and
    will be switched to the EIP gas price model at block 2,463,000
 
-Issues to address:
+## Issues to address:
 
 1. The need to avoid silverware drawer of ethereums:
   * EF Official
@@ -33,7 +33,7 @@ Issues to address:
 2. Urgency - this needs to be ready for testing as soon as humanly possible.
 3. MAKE THE PLAN KNOWN. on all channels. smoke signals as need be.
 
-Proposed actions:
+## Proposed actions:
 
 1. MAKE KNOWN TO EVERYONE WHAT THE DECISION IS AND PUBLISH REQUIRED CHANGES IN ACCEPTED ECIP. 
 2. Incorporate EIP-150 as ECIP-1015, to follow our process
@@ -42,3 +42,9 @@ Proposed actions:
 5. Implement ECIP-1015 on top of our 2.0.0 geth code base, test it
 6. Release new classic geth version, promote its usage heavily among ETC users, as well as with pools, exchanges, and wallets that support ETC
 7. Hard fork on block 2,500,000 ~2016-10-25
+
+## License
+
+This work is licensed under [Apache License, Version
+2.0](http://www.apache.org/licenses/).
+
